dor-services.gemspec

```diff
@@ -47,7 +47,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'lyber-utils', '~> 0.1.2'
   s.add_dependency 'moab-versioning', '~> 1.4.4' # 1.3.2 fails, 1.4.3 fails
   s.add_dependency 'modsulator', '~> 0.0.7'
-  s.add_dependency 'stanford-mods', '= 1.3.3' # 1.3.4 fails on pub_date_sort, 1.4.0 fails
+  s.add_dependency 'stanford-mods', '= 1.5.3'  # v1.5.4 changes what sw_genre returns, which breaks test expectations.  TODO: figure out how to keep argo indexed counterparts of searchworks fields in sync w/ current SW behavior.
```

Comment:
the SearchWorks sprint is indeed changing stuff, and we need to be better about our semantic versioning.  I've goofed it up (1.3.4 should have been 1.4.0), and I forgot to tell @lmcglohon  to be sure to follow semantic versioning, too.  Sorry!   

the sw_genre values have changed;   Glancing thru the closed PRs or the recent commits is probably a fast way to figure out what happened.